src/sql/magic.py

```diff
@@ -640,6 +674,38 @@ def _persist_dataframe(
         )
 
 
+def get_query_type(command: str):
+    """
+    Returns the query type of the original sql command
+    """
+    query_type = (
+        sqlparse.parse(command)[0].get_type() if sqlparse.parse(command) else None
```

Comment:
this is inefficient because it'll run sqlparse.parse twice, better to execute it only once

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/files#diff-527caf90dd08297f5253e826bd98fea0b6f4fbd44fe9f20472976f77868dac06R677-R678)